bugfix: when loop local directories, listObjects use request's prefix…

djyin4op opened the pull request:
… now .

Commit:
bugfix: when loop local directories, listObjects use request's prefix now .

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/aliyun/oss-emulator?pullRequest=4) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/aliyun/oss-emulator?pullRequest=4) before we can accept your contribution.<br/><hr/>**菜地** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/aliyun/oss-emulator?pullRequest=4) it.</sub>

Commit:
bugfix: when loop local directories, listObjects use request's prefix now .
and delete test code.

Commit:
bugfix:
WEBrick's log is not quite in Win2008R2.
change log's file location, from ../log --> ./log

Commit:
bugfix:
can not list buckets
